extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -425,6 +445,38 @@ export class AmpStoryPage extends AMP.BaseElement {
     return Promise.all(mediaPromises);
   }
 
+  /**
+   * Finds embedded components in page and prepares them for their expanded view
+   * animation.
+   * @param {number} timeout
+   * @private
+   */
+  findAndPrepareEmbeddedComponents_(timeout) {
```

Comment:
This method doesn't need to get the timeout as a parameter, it should be accessible directly.